Apps/WebClient/src/ClientApp/src/components/private/sharing/DelegateInvitationDialog.vue

```diff
@@ -0,0 +1,186 @@
+const currentTitle = computed(() => {
+    switch (delegateStore.invitationWizardState?.step) {
+        case DelegateInvitationWizardStep.contact:
+            return "Invite Someone";
+        case DelegateInvitationWizardStep.dataSources:
+            return "Select Record Types";
+        case DelegateInvitationWizardStep.expiryDate:
+            return "Select Access Expiry";
+        case DelegateInvitationWizardStep.review:
+            return "Review Invitation";
+        case DelegateInvitationWizardStep.sharingCode:
+            return "Sharing Code";
+        default:
+            return "Invite Someone";
+    }
+});
+const nextStepButtonTitle = computed(() => {
+    switch (delegateStore.wizardNextStep) {
+        case DelegateInvitationWizardStep.dataSources:
+            return "Next: Select Records";
+        case DelegateInvitationWizardStep.expiryDate:
+            return "Next: Expiry Date";
+        case DelegateInvitationWizardStep.review:
+            return "Next: Review";
+        case DelegateInvitationWizardStep.sharingCode:
+            return "Next: Sharing Code";
+        default:
+            return "Done";
+    }
+});
+
+function handleCancel(): void {
+    delegateStore.clearInvitationDialogState();
+}
+
+function handleBack(): void {
+    delegateStore.wizardToPreviousStep();
+}
+
+function handleSaveCurrentStep(): void {
+    switch (currentStep.value) {
+        case DelegateInvitationWizardStep.contact:
+            invitationContactStepComponent.value?.saveStep();
+            break;
+        case DelegateInvitationWizardStep.dataSources:
+            invitationRecordTypesStepComponent.value?.saveStep();
+            break;
+        case DelegateInvitationWizardStep.expiryDate:
+            invitationExpiryStepComponent.value?.saveStep();
+            break;
+        case DelegateInvitationWizardStep.review:
+            invitationReviewStepComponent.value?.saveStep();
+            break;
+        case DelegateInvitationWizardStep.sharingCode:
+            delegateStore.clearInvitationDialogState();
+            break;
+        default:
+            break;
+    }
+}
+</script>
+
+<template>
+    <v-row justify="center">
+        <v-dialog v-model="showDialog" max-width="800px">
+            <v-card data-testid="delegation-invitate-dialog">
+                <v-card-title class="bg-primary px-0">
+                    <v-toolbar
+                        :title="currentTitle"
+                        density="compact"
+                        color="primary"
+                    >
+                        <HgIconButtonComponent
+                            data-testid="invitation-dialog-close-button"
+                            icon="close"
+                            aria-label="Close"
+                            @click="handleCancel"
+                        />
+                    </v-toolbar>
+                </v-card-title>
+                <v-card-text v-if="currentStep !== undefined" class="pb-4">
```

Comment:
Standard padding on dialog card text is `pa-4`.

Reply:
Cool it made a difference before adding mt-4 on the actions